### PR TITLE
Update AddLanguage.jsx

### PR DESCRIPTION
**Pull request by manumoreira**
Delete filter for code set for 639-2/B not null to add more languages like Lugbara, Teso, Ganda, Nkore, and Nyankole

### CODE REVIEW

**Review by matiasgarciaisaia — COMMENTED** [review]
I was wandering if the null check was added for something code-wise, but it seems to have been a decision about how to filter-out languages ([comment](https://github.com/instedd/surveda/issues/356#issuecomment-540117379)).

It'd be nice to run the code to actually check nothing else breaks, though.

File: assets/js/components/questionnaires/AddLanguage.jsx [hunk 1]
```diff
@@ -33,7 +33,7 @@ class AddLanguage extends Component {
       lang.text.indexOf(value) !== -1 || lang.text.toLowerCase().indexOf(value) !== -1
 
     let languagesOptions = iso6393
-      .filter((lang) => lang.type == "living" && lang.iso6392T != null)
+      .filter((lang) => lang.type == "living")
       .map((lang) => ({ id: idForLanguange(lang), text: lang.name }))
```
Comment (matiasgarciaisaia) [inline assets/js/components/questionnaires/AddLanguage.jsx:37]: From [the mapping function](https://github.com/instedd/surveda/blob/9b405132d00d5c849d5081055ebfcd09a0c61ea3/assets/js/components/questionnaires/AddLanguage.jsx#L31), it shows we're not using the `iso6392T` for anything else than trimming down the languages list (exactly what we're reverting with this PR), so the null-check doesn't change a thing in the app's behaviour.

**Review by matiasgarciaisaia — APPROVED** [review]
Looks good! 👍 :shipit:

**Review by aeatencio — APPROVED** [review]
Working OK in my local 👍

**Pull request merged**